seqr/views/utils/terra_api_utils.py

```diff
@@ -67,6 +69,12 @@ def anvil_call(method, path, access_token, user=None, headers=None, root_url=Non
     url, headers = _get_call_args(path, headers, root_url)
     request_func = getattr(requests, method)
     headers.update({'Authorization': 'Bearer {}'.format(access_token)})
+    cache_key = 'terra_req__' + hashlib.md5((url + str(headers) + str(kwargs)).encode('utf-8')).hexdigest()
+    r_text = safe_redis_get_json(cache_key)
+    if r_text:
+        logger.info('Debug: Cache {} hit. {}'.format(cache_key, r_text))
+        return json.loads(r_text)
```

Comment:
`safe_redis_get_json` should return it as loaded json so this should be unnecessary/ shouldn't actually work

Reply:
Really?! I should add a test to cover it. In the real code scenarios that I tried, no caches were used.

Reply:
If no caches are used then why on earth would we add this code??? Adding untested code we expect to never be used is ridiculous! There are multiple tests that assert that in a given single API call we make multiple identical calls to Terra! In all of those use cases I would absolutely expect us to be using the cache, and in fact that is the entire reason to write this code

Reply:
I found that there are some tests that cover it. And I found that `json.loads()` works with a `dict` parameter. That's why I didn't catch the mistake.
There are 4 API calls, one for validation of account registration on AnVIL while logging in, one for workspace list when displaying the dashboard, one for access permissions of a workspace, one for the collaborator list of a workspace. The one that could be called often is the one for access permissions.
The reason that I didn't see cache hits yesterday is that I use an account with local permissions. After I switch my account, I can see multiple cache hits when calling the API for workspace access permissions.